paas/router.j2

```diff
@@ -10,6 +10,7 @@
 
       DM_API_URL: 'https://dm-api-{{ environment }}.cloudapps.digital'
       DM_SEARCH_API_URL: 'https://dm-search-api-{{ environment }}.cloudapps.digital'
+      DM_ANTIVIRUS_API_URL: 'https://dm-antivirus-api-{{ environment }}.cloudapps.digital'
```

Comment:
You'll probably want to add this var to  the `Makefile` nginx linting command as well (and maybe `docker-compose.yml.example`?)

Reply:
Wait those are in the router repo. I've got myself confused 😸